pxr/usd/usdGeom/mesh.h

```diff
@@ -253,7 +253,7 @@ class UsdGeomMesh : public UsdGeomPointBased
     // FACEVERTEXCOUNTS 
     // --------------------------------------------------------------------- //
     /// Provides the number of points in each face of the mesh, 
```

Comment:
Ditto to previous comment re: points vs vertices

Reply:
Noted -- the combination of "face" and "vertex" clearly makes for "face-vertex", "face" and "point" do not.